src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {Pressable} from 'react-native';
+import * as anchorForAttachmentsOnlyPropTypes from './anchorForAttachmentsOnlyPropTypes';
+import AttachmentView from '../AttachmentView';
+import fileDownload from '../../libs/fileDownload';
+import addEncryptedAuthTokenToURL from '../../libs/addEncryptedAuthTokenToURL';
+
+class BaseAnchorForAttachmentsOnly extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isDownloading: false,
+        };
+        this.processDownload = this.processDownload.bind(this);
+    }
+
+    /**
+     * Initiate file downloading and update downloading flags
+     *
+     * @param {String} href
+     * @param {String} fileName
+     */
+    processDownload(href, fileName) {
+        this.setState({isDownloading: true});
+        fileDownload(href, fileName).then(() => this.setState({isDownloading: false}));
+    }
+
+    render() {
+        const source = addEncryptedAuthTokenToURL(this.props.source);
```

Comment:
We shouldn't add encyptToken for local file:
More details: https://github.com/Expensify/App/issues/54640

Reply:
Yes, you're correct. My bad, just found the offending PR.